Northstar.CustomServers/mod/scripts/vscripts/sh_progression.nut

```diff
@@ -88,21 +88,22 @@ bool function ClientCommand_SetProgression( entity player, array<string> args )
 
 /// Resets a specific Titan's Aegis rank back to `0`
 /// * `player` - The player entity to perform the action on
-/// * `args` - The arguments passed from the client command. `args[0]` should be an integer corresponding to the index of the Titan to reset.
+/// * `args` - The arguments passed from the client command. `args[0]` should be a string corresponding to the chassis name of the Titan to reset.
 ///
 /// Returns `true` on success and `false` on missing args.
 bool function ClientCommand_ResetTitanAegis( entity player, array<string> args )
 {
 	if ( !args.len() )
 		return false
 	
-	int suitIndex = args[0].tointeger()
+	string titanRef = args[0]
+	int suitIndex = PersistenceGetEnumIndexForItemName( "titanClasses", titanRef )
```

Comment:
This will cause a script error if it is invalid afaik, you should validate that it is a valid ref by doing `PersistenceEnumValueIsValid( "titanClasses", titanRef )`

Reply:
Also done.